backend/api/src/main/java/com/virtuslab/gitmachete/backend/api/NullGitMacheteRepositorySnapshot.java

```diff
@@ -39,8 +40,8 @@ public List<IManagedBranchSnapshot> getManagedBranches() {
   }
 
   @Override
-  public Option<IManagedBranchSnapshot> getManagedBranchByName(String branchName) {
-    return Option.none();
+  public @Nullable IManagedBranchSnapshot getManagedBranchByName(@Nullable String branchName) {
```

Comment:
Hmmm can it be just
```suggestion
  public @Nullable IManagedBranchSnapshot getManagedBranchByName(String branchName) {
```

Reply:
The definition in the base interface (`com.virtuslab.gitmachete.backend.api.IGitMacheteRepositorySnapshot#getManagedBranchByName`) doesn't allow for `branchName` to be nullable

Reply:
This doesn't break the contract (as in general, an implementing class can **relax** its interface's contract), but seems redundant anyway